dmv/test/dmv_test.js

```diff
@@ -41,7 +53,9 @@ describe('DMV', () => {
 
     dmv.enter("Skidding Skye");
     dmv.enter("Drivin' Dave");
-    dmv.nextCustomer();
```

Comment:
setting the `let` for the function before double calling as parameters is a better option.

Reply:
Please provide another comment to demonstrate "Speak fluently using object-oriented / coding terminology; take code and describe it using words like method, parameter, declare, instantiate...etc"